frontend/src/app/projects/project-detail/model-overview/model-overview.component.html

```diff
@@ -4,8 +4,9 @@
  -->
 
 <div>
-  <div class="flex items-center">
-    <h2>Models</h2>
+  <div class="mt-1.5 flex items-center">
+    <h2 class="text-xl font-medium">Models</h2>
+    <h2 class="text-xl font-medium">Models</h2>
```

Comment:
The `<h2>` is added twice: 

![image](https://github.com/DSD-DBS/capella-collab-manager/assets/23395732/c8f65c89-e4dc-42dc-bb0e-aa3ebcfd0fba)